extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```diff
@@ -1573,6 +1573,15 @@ describes.realWin('amp-ad-network-doubleclick-impl', realWinConfig, (env) => {
       expect(impl.getSafeframePath()).to.match(new RegExp(expectedPath));
     });
 
+    it('should use the same random subdomain for every slot on a page', () => {
+      impl.experimentIds = [RANDOM_SUBDOMAIN_SAFEFRAME_BRANCHES.EXPERIMENT];
+
+      const first = impl.getSafeframePath();
+      const second = impl.getSafeframePath();
+
+      expect(first).to.equal(second);
```

Comment:
Would this have failed before?

Reply:
Good catch. Using the same impl instance means it is the same slot. 

Just changed it to make a different instance of AmpAdNetworkDoubleclickImpl, which would be the same as making another slot. (With this new test change it would have failed before).